Author: Fabian Fagerholm <fabbe@debian.org>
Description: Rename the testsuite program to sasltestsuite and use /etc/sasldb2
instead of ./sasldb as default path for the sasldb database file.
--- a/utils/testsuite.c
+++ b/utils/testsuite.c
@@ -462,9 +462,9 @@ int good_getopt(void *context __attribut
 	    *len = (unsigned) strlen("sasldb");
 	return SASL_OK;
     } else if (!strcmp(option, "sasldb_path")) {
-	*result = "./sasldb";
+	*result = "/etc/sasldb2";
 	if (len)
-	    *len = (unsigned) strlen("./sasldb");
+	    *len = (unsigned) strlen("/etc/sasldb2");
 	return SASL_OK;
     } else if (!strcmp(option, "canon_user_plugin")) {
 	*result = cu_plugin;
@@ -2924,7 +2924,7 @@ void notes(void)
 void usage(void)
 {
     printf("Usage:\n" \
-           " testsuite [-g name] [-s seed] [-r tests] -a -M\n" \
+           " sasltestsuite [-g name] [-s seed] [-r tests] -a -M\n" \
            "    g -- gssapi service name to use (default: host)\n" \
 	   "    r -- # of random tests to do (default: 25)\n" \
 	   "    a -- do all corruption tests (and ignores random ones unless -r specified)\n" \
